src/org/freedesktop/gstreamer/lowlevel/GstVideoAPI.java

```diff
@@ -109,4 +108,30 @@ public GstVideoTimeCodeConfigStruct(Pointer p) {
             read();
         }
     }
+    
+    @Structure.FieldOrder({
+    	"meta", "buffer", "flags",
+    	"videoFormat", "id", "width", "height", "n_planes", 
+        "offset", "stride", 
+        "map", "unmap",
+        "alignment"})
+    class GstVideoMetaStruct extends Structure {
+    	public GstMetaAPI.GstMetaStruct.ByValue meta;
```

Comment:
Isn't structure in structure by value anyway?

Reply:
Exact
I have made a copy of code GstVideoTimeCodeMetaStruct 
I have modified both structures to suppress "ByValue"